### PR TITLE
add jacoco

### LINKED ISSUE #169: Add code coverage tool
## Description
Add code coverage tool for Android tests

## Steps
- [ ] Add jacoco

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**
closes #169

### CODE REVIEW

File: gradle/jacoco.gradle [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        def coverageTaskName = "${testTaskName}Coverage"
+
+        // Create coverage task of form 'testFlavorTypeUnitTestCoverage' depending on 'testFlavorTypeUnitTest'
+        task "${coverageTaskName}"(type: JacocoReport, dependsOn: "$testTaskName") {
+            group = 'Reporting'
+            description = "Generate Jacoco coverage reports for the ${sourceName.capitalize()} build."
+
+            classDirectories = fileTree(
+                    dir: "${project.buildDir}/intermediates/javac/${sourcePath}",
+                    excludes: ['**/R.class',
```
Comment (RingerJK) [inline gradle/jacoco.gradle]: shouldn't we exclude libraries `mapbox-telemetry`, `OkHttp`, `*mockk*` and etc?

Comment (alexander-kulikovskii) [reply gradle/jacoco.gradle]: only if library added as source code

File: gradle/jacoco.gradle [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            executionData = files("${project.buildDir}/jacoco/${testTaskName}.exec")
+
+            reports {
+                xml.enabled = true
+                html.enabled = true
+            }
+        }
+        build.dependsOn "${coverageTaskName}"
+    }
+}
```
Comment (yunikkk) [inline gradle/jacoco.gradle]: We don't yet expose jacoco collected stats, right?

Comment (alexander-kulikovskii) [reply gradle/jacoco.gradle]: yes

Comment (yunikkk) [reply gradle/jacoco.gradle]: Mind creating a task to do that?

**Pull request merged**